.circleci/config.yml

```diff
@@ -335,10 +334,63 @@ jobs:
             https://api.github.com/repos/NCAR/VAPOR/issues/${CIRCLE_PULL_REQUEST##*/} \
             -d "$json"
 
-  build_ubuntu18:
+  python_api_centos:
+    docker:
+      - image: sgpearse/vapor3-centos7:latest
+
+    resource_class: large
+
+    steps:
+      - run:
+          name: update 3rd party libraries
```

Comment:
The conda build does not use the third party libraries.

Reply:
Huh I thought I commented here.  Fixed.